src/Engine/McBopomofoLM.h

```diff
@@ -107,6 +107,7 @@ class McBopomofoLM : public Formosa::Gramambular2::LanguageModel {
 
     /// Sets a lambda to convert the macro to a string.
     void setMacroConverter(std::function<std::string(const std::string&)> macroConverter);
+    std::string convertMacro(std::string  input);
```

Comment:
`const std::string& input`.

Reply:
Fixed in 92e46601d055fb57699128d5d2cb8634a1fef339